content/kotlin/concepts/strings/terms/lowercase/lowercase.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.lowercase()'
+Description: 'The lowercase method in Kotlin is used to convert the characters of a string to lowercase.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The `.lowercase()` method in Kotlin is used to convert the characters of a string to lowercase. 
+
+## Syntax
+
+```pseudo
+fun String.lowercase(locale: Locale = Locale.getDefault()): String
+```
+
+- `locale` (optional): A `Locale` object representing the language and region-specific rules for case conversion. The default value is `Locale.getDefault()`.
+
+## Examples
+
+Let's dive into some examples to see how the `.lowercase()` method can be used:
+
+```kotlin
+fun main() {
+    println("Iced frappé!".lowercase())
```

Comment:
This example is a direct copy of an example in the Kotlin documentation. This is considered plagiarism, and is not permitted within docs. All docs content must be original. Revise the example with an example you have developed